front-end/src/components/Users/register.js

```diff
@@ -53,7 +53,7 @@ const RegistrationForm = () => {
         const { displayName } = airtableInfo.data;
 
         // at this point we know the user is in the airtable
-        if (emailDomain === "santropolroulant.org") {
+        if (emailDomain === "santropolroulant.org" || emailDomain === "softwareforlove.com") {
```

Comment:
Shouldn't this just be santropolroulant.org in Prod?

Reply:
I think this branch started from an old version because the main branch only checks if the email domain is santropolroulant.org